db/migrate/20210223173201_add_gender_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddGenderToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :gender, :integer
```

Comment:
We can add a not null restriction here. WDYT?

Reply:
I mean, I could but I think it would be redundant:
`validates :gender, presence: true, inclusion: { in: genders.keys, message: :invalid }`
This already validates its presence. If you'd like me to add it anyways in order to have a database restriction then I could

Reply:
It is nice to have dababase restrictions. 
In case database is updated via SQL

Reply:
👍

Reply:
+ 1 for DB constraints and validations. You. can have race conditions on your Ruby code but the DB handles concurrency much betterr